src/components/home/elements/TokenRecord.vue

```diff
@@ -371,33 +377,60 @@ export default {
             this.$root.$emit('bv::show::modal', 'tokenInfoModal_' + this.tokenId)
         },
         sendToken() {
-            this.getTokenBalances()
-            this.getTokenInfo()
-            this.$root.$emit('bv::show::modal', 'sendTokenModal_' + this.tokenId)
+            if (this.getDevice && this.getDevice === 'Ledger') {
+                confirm('This feature is not supported')
+            } else {
+                this.getTokenBalances()
+                this.getTokenInfo()
+                this.$root.$emit('bv::show::modal', 'sendTokenModal_' + this.tokenId)
+            }
         },
         supersede() {
-            this.getTokenInfo()
-            this.$root.$emit('bv::show::modal', 'supersedeModal_' + this.tokenId)
+            if (this.getDevice && this.getDevice === 'Ledger') {
```

Comment:
no need check `this.getDevice` is null because it always exist. Simplify the logic to `if (this.getDevice === 'Ledger')`